lib/configs/flat/base.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * IMPORTANT!
+ * This file has been automatically generated,
+ * in order to update its content execute "npm run update"
+ */
+const globals = require('globals')
+const vueEslintParser = require('vue-eslint-parser')
+module.exports = {
+  languageOptions: {
+    parser: vueEslintParser,
+    globals: {
+      ...globals.browser,
+      ...globals.es2015
+    },
+    parserOptions: {
+      ecmaVersion: 2020,
+      sourceType: 'module'
+    }
```

Comment:
In most cases, you don't need to specify `parserOptions`. just use:

```js
  languageOptions: {ecmaVersion: 2020, sourceType: 'module'}
```
eslint will pass it to the parser. So you don't need to specify global.es2015 either.

Reply:
https://eslint.org/docs/next/use/configure/language-options#specifying-javascript-options